tooling/api/src/main/java/org/jboss/windup/tooling/IOptionKeys.java

```diff
@@ -0,0 +1,14 @@
+package org.jboss.windup.tooling;
+
+public interface IOptionKeys 
+{
+	public static final String sourceModeOption = "sourceMode";
```

Comment:
What do you think on following SUN's naming convention about constants ? https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-135099.html#367